SecurePlayerSampleApp/app/src/main/java/com/ooyala/sample/players/SecurePlayerEHLSPlayerActivity.java

```diff
@@ -102,6 +107,11 @@ public void update(Observable arg0, Object argN) {
       return;
     }
 
+    // Automation Hook: to write Notifications to a temporary file on the device/emulator
+    String text="Notification Received: " + arg1 + " - state: " + player.getState();
+    // Automation Hook: Write the event text along with event count to log file in sdcard if the log file exists
+    playbacklog.writeToSdcardLog(text);
+
     Log.d(TAG, "Notification Received: " + arg1 + " - state: " + player.getState());
```

Comment:
Use the `text` variable in the `Log`. Let's not use two different with the same stuff. 

Please do that for all the Activities that applied the same change :)